### ISSUE TITLE
Provide a possibility to set up elasticsearch_password

### ISSUE THREAD

**Issue by andrewnazarov** [elastic/terraform-provider-ec#641]
## Overview
Right now the elasticsearch_user and elasticsearch_password are generated only once when the instance is created. We can output them and use them for further set-up or in the other pieces of code. However, if we want to change the password this would be no longer valid. The same goes for the migration when we export and import resources (which happened recently in the latest version of this provider). The idea is to have the possibility to set up this password explicitly. Then we could take the value from external sources (either from Vault or from Google Secret Manager) and provide it to the EC instance.

It would make sense to do the same for an elasticsearch_user. Though it's not that crucial.

**Comment by tobio** [comment]
Whilst it would be nice to explicitly manage the password in Terraform it's unlikely to be implemented in the underlying API. In reality, this is a duplicate of https://github.com/elastic/terraform-provider-ec/issues/99.
